python/lsst/ts/atspectrograph/atspec_csc.py

```diff
@@ -734,33 +699,23 @@ async def home_element(
 
     def assert_move_allowed(self, action: str) -> None:
         """Assert that moving the spectrograph elements is allowed."""
-        if self.is_exposing:
-            raise salobj.base.ExpectedError(
-                f"Camera is exposing, {action} is not allowed."
-            )
+        if not self.not_exposing.is_set():
+            raise salobj.base.ExpectedError(f"Camera is exposing, {action} is not allowed.")
```

Comment:
Suggest for consistency using `salobj.ExpectedError`.
```suggestion
            raise salobj.ExpectedError(f"Camera is exposing, {action} is not allowed.")
```